### PR TITLE
DCOS-8430: Rename State to Status in TaskTable

### PR DESCRIPTION
**Pull request by daiyiwhale**
Rename State to Status in TaskTable, for consistency across application.

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @orlandohohmeier

File: src/js/components/TaskTable.js [hunk 1]
```diff
@@ -30,8 +30,8 @@ class TaskTable extends React.Component {
     return task.resources[prop];
   }
 
-  getStateValue(task, prop) {
-    return TaskStates[task[prop]].displayName;
+  getStatusValue(task) {
+    return TaskStates[task['state']].displayName;
```
Comment (jfurrow) [inline src/js/components/TaskTable.js]: We can just use `task.state` here

Comment (pierlo-upitup) [inline src/js/components/TaskTable.js]: or we could even `getStatusValue({state}) { return TaskStates[state].displayName; }` if we wanted to be fancy

**Comment by pierlo-upitup** [comment]
Relevant tests are passing
image

File: src/js/components/TaskTable.js (hunk 1 above)
Comment (daiyiwhale) [inline src/js/components/TaskTable.js]: done!

**Pull request merged**